<html xmlns:string="xalan://java.lang.String" xmlns:lxslt="http://xml.apache.org/xslt">
<head>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Unit Test Results: All Errors</title>
<link rel="stylesheet" type="text/css" title="Style" href="stylesheet.css">
</head>
<body onload="open('allclasses-frame.html','classListFrame')">
<h1>Unit Test Results.</h1>
<table width="100%">
<tr>
<td align="left"></td><td align="right">Designed for use with <a href="http://www.junit.org/">JUnit</a> and <a href="http://ant.apache.org/">Ant</a>.</td>
</tr>
</table>
<hr size="1">
<h2>All Errors</h2>
<table class="details" border="0" cellpadding="5" cellspacing="2" width="95%">
<tr valign="top">
<th>Class</th><th>Name</th><th>Status</th><th width="80%">Type</th><th nowrap>Time(s)</th>
</tr>
<tr valign="top" class="Error">
<td><a href="ptolemy/actor/lib/jjs/modules/serial/test/junit/64_JUnitTclTest.html">JUnitTclTest</a></td><td><a name="[3] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/serial/test/auto/SerialLoopbackNumber.xml (RunModel)"></a><a href="ptolemy/actor/lib/jjs/modules/serial/test/junit/64_JUnitTclTest.html#[3] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/serial/test/auto/SerialLoopbackNumber.xml (RunModel)">[3] /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/modules/serial/test/auto/SerialLoopbackNumber.xml (RunModel)</a></td><td>Error</td><td>N/A<br>
<br>
<code>java.lang.reflect.InvocationTargetException<br>	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)<br>	at ptolemy.util.test.junit.AutoTests.RunModel(AutoTests.java:193)<br>	at junitparams.internal.InvokeParameterisedMethod.evaluate(InvokeParameterisedMethod.java:102)<br>Caused by: ptolemy.kernel.util.IllegalActionException: Failure executing the react function: Error: commonHost.js, react(), invoking a specific handler for "input": Exception occurred in input handler for accessor TrainableTest, which has now has been removed.  Exception was: Error: TrainableTest: After seeing 1 tokens, the input "2" is not within "1e-9" of the expected value "1" Stacktrace was: Error: TrainableTest: After seeing 1 tokens, the input "2" is not within "1e-9" of the expected value "1"<br>	at &lt;anonymous&gt; (&lt;function&gt;:267)<br>	at invokeSpecificHandler (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1724)<br>	at react (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1769) in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt; at line number 1736 at column number 24<br>  in .SerialLoopbackNumber.TrainableTest<br>Because:<br>Error: commonHost.js, react(), invoking a specific handler for "input": Exception occurred in input handler for accessor TrainableTest, which has now has been removed.  Exception was: Error: TrainableTest: After seeing 1 tokens, the input "2" is not within "1e-9" of the expected value "1" Stacktrace was: Error: TrainableTest: After seeing 1 tokens, the input "2" is not within "1e-9" of the expected value "1"<br>	at &lt;anonymous&gt; (&lt;function&gt;:267)<br>	at invokeSpecificHandler (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1724)<br>	at react (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1769) in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt; at line number 1736 at column number 24<br>	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2306)<br>	at ptolemy.actor.lib.jjs.JavaScript.fire(JavaScript.java:1138)<br>	at ptolemy.domains.de.kernel.DEDirector._fire(DEDirector.java:1949)<br>	at ptolemy.domains.de.kernel.DEDirector.fire(DEDirector.java:502)<br>	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)<br>	at ptolemy.actor.Manager.iterate(Manager.java:851)<br>	at ptolemy.actor.Manager.execute(Manager.java:369)<br>	at ptolemy.actor.Manager.run(Manager.java:1263)<br>	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)<br>Caused by: javax.script.ScriptException: Error: commonHost.js, react(), invoking a specific handler for "input": Exception occurred in input handler for accessor TrainableTest, which has now has been removed.  Exception was: Error: TrainableTest: After seeing 1 tokens, the input "2" is not within "1e-9" of the expected value "1" Stacktrace was: Error: TrainableTest: After seeing 1 tokens, the input "2" is not within "1e-9" of the expected value "1"<br>	at &lt;anonymous&gt; (&lt;function&gt;:267)<br>	at invokeSpecificHandler (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1724)<br>	at react (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1769) in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt; at line number 1736 at column number 24<br>	at jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:470)<br>	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:392)<br>	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeMethod(NashornScriptEngine.java:199)<br>	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2281)<br>Caused by: /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1736:24 Error: commonHost.js, react(), invoking a specific handler for "input": Exception occurred in input handler for accessor TrainableTest, which has now has been removed.  Exception was: Error: TrainableTest: After seeing 1 tokens, the input "2" is not within "1e-9" of the expected value "1" Stacktrace was: Error: TrainableTest: After seeing 1 tokens, the input "2" is not within "1e-9" of the expected value "1"<br>	at &lt;anonymous&gt; (&lt;function&gt;:267)<br>	at invokeSpecificHandler (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1724)<br>	at react (/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1769)<br>	at jdk.nashorn.internal.objects.NativeError.initException(NativeError.java:137)<br>	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:102)<br>	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:106)<br>	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:110)<br>	at jdk.nashorn.internal.objects.NativeError.constructor(NativeError.java:129)<br>	at jdk.nashorn.internal.scripts.Script$Recompilation$12474$76478A$require.L:1#react#invokeSpecificHandler(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1741)<br>	at jdk.nashorn.internal.scripts.Script$Recompilation$12473$75887A$require.L:1#react(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#409:30&lt;eval&gt;:1769)<br>	at jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:639)<br>	at jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:494)<br>	at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:393)<br>	at jdk.nashorn.api.scripting.ScriptObjectMirror.callMember(ScriptObjectMirror.java:199)<br>	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:386)<br>
</code></td><td>15.839</td>
</tr>
</table>
</body>
</html>
